Compiler/runtime/systemimpl.c

```diff
@@ -2230,6 +2230,7 @@ extern char* SystemImpl__iconv(const char * str, const char *from, const char *t
   iconv_t ic;
   int count;
   char *buf;
+  if (!strcmp(from, to)) return (char*)str;
```

Comment:
This is wrong. iconv from UTF-8 to UTF-8 does the magical operation: verify that the input is in UTF-8.